core/services/job/spawner.go

```diff
@@ -0,0 +1,186 @@
+package job
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"sync"
+
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/services/eth"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+)
+
+// The job spawner manages the spinning up and spinning down of the long-running
+// services that perform the work described by job specs.  Each active job spec
+// has 1 or more of these services associated with it.
+//
+// At present, Flux Monitor and Offchain Reporting jobs can only have a single
+// "initiator", meaning that they only require a single service.  But the older
+// "direct request" model allows for multiple initiators, which imply multiple
+// services.
```

Comment:
Im not clear on what this means. Is everything being brought under umbrella of "Spawner"?

Reply:
Yeah, this is part of my drive to create a bunch of reusable components that we can gradually implement for all request models.